challenges/classes.js

```diff
@@ -1,7 +1,63 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
-
+//function CuboidMaker(cadd) {
+ //   this.length = cadd.length;
+ //   this.width = cadd.width;
+ //   this.height = cadd.height;
+ // }
+  class CuboidMaker{
+      constructor(cadd){
+        this.length = cadd.length;
+        this.width = cadd.width;
+        this.height = cadd.height;
+      }
+  
+  
+  /* == Step 2: Volume Method ==
+    Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
+    
+    Formula for cuboid volume: length * width * height
+  */
+  //CuboidMaker.prototype.volume = function() {
+  //  return this.length * this.width * this.height;
+  //};
+  
+  volume(){
+    return `${this.length} * ${this.width} * ${this.height}`;
+  }
+  
+  /* == Step 3: Surface Area Method ==
+    Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
+  
+    Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
+  */
+  //CuboidMaker.prototype.surfaceArea = function() {
+    //return (2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+    //);
+      //};
+  surfaceArea(){
+    return `2 * ${this.length} * ${this.width} + ${this.length} * ${this.height} + ${this.width} * ${this.height}`;
+  }
+  }
+  /* == Step 4: Create a new object that uses CuboidMaker ==
+    Create a cuboid object that uses the new keyword to use our CuboidMaker constructor
+    Add properties and values of length: 4, width: 5, and height: 5 to cuboid.   
+  */
+ class card extends CuboidMaker{
+    constructor(caddi){
+        super(caddi)
+        this.length = caddi.length,
+        this.width = caddi.width,
+        this.weight = caddi.weight
+  }
+}
+  const cuboid = new CuboidMaker({
+    length: 4,
+    width: 5,
+    height: 5
+  });
+  
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+ console.log(cuboid.volume()); // 100
+ console.log(cuboid.surfaceArea()); // 130
```

Comment:
Review your `return` statements on your methods. They are returning a `string`. They are supposed to return a value.